lib/model/database.g.dart

```diff
@@ -133,7 +384,6 @@ class $AccountsTable extends Accounts with TableInfo<$AccountsTable, Account> {
     if (data.containsKey('id')) {
       context.handle(_idMeta, id.isAcceptableOrUnknown(data['id']!, _idMeta));
     }
-    context.handle(_realmUrlMeta, const VerificationResult.success());
```

Comment:
nit: generated code should update in tandem with the deps upgrade that causes it